plugins/code_highlight.ts

```diff
@@ -30,6 +33,13 @@ export default function (userOptions?: Partial<Options>) {
   // @ts-ignore: Property 'configure' does not exist on type '{}'
   hljs.configure(options.options);
 
+  if (userOptions && userOptions.languages) {
```

Comment:
`userOptions` is merged into `options` variable, so this could be simplified with:

```suggestion
  if (options.languages) {
```